examples/config-overrides.js

```diff
@@ -14,6 +14,11 @@ module.exports = function override(config) {
   config.plugins.push(
     new CopyWebpackPlugin({
       patterns: [
+        {
+          from: `${revealSource}/**/reveal.*.worker.js`,
```

Comment:
Isn't it handled already by next pattern `*.worker.js` ?

Reply:
You're right, this was the result of some back and forth changes